src/lib/mongoBackend/mongoRegisterContext.cpp

```diff
@@ -52,13 +52,10 @@ HttpStatusCode mongoRegisterContext
   std::map<std::string, std::string>&  uriParam,
   const std::string&                   tenant,
   const std::string&                   servicePath
-
 )
 {
-    // FIXME P10: we have been discussing about changing the "notifyFormat" name for something more neutral and more
-    // meaningfull for *both* subscription notification and cpr forwards
-    std::string notifyFormat = uriParam[URI_PARAM_NOTIFY_FORMAT];
-    std::string sPath = servicePath;
+    std::string  sPath         = servicePath;
+    std::string  notifyFormat  = uriParam[URI_PARAM_NOTIFY_FORMAT];
```

Comment:
Anyway, now we have in some places:

```
std::string notifyFormat = uriParam[URI_PARAM_NOTIFY_FORMAT];
```

while in others:

```
const std::string notifyFormat = uriParam[URI_PARAM_NOTIFY_FORMAT];
```

Should we be homogenous and use always the same "style"?